### DISCUSSION TITLE
Custom upgrade hook?

### DISCUSSION THREAD

**Discussion by comminutus** [89luca89/distrobox#1569] [Q&A]
Hello, I have a customized container image that I use with distrobox.  Some packages that I install aren't provided by package managers and I have to install them manually (usually with curl, etc).  When `distrobox upgrade` runs and updates the container's packages, is there some way to run a custom upgrade script that would upgrade these manually installed packages too?

**Comment by skymoo** [comment]
I was after something like this as well, specifically I have an Arch container with some things installed from the AUR and `distrobox upgrade` doesn't touch AUR packages?
